scripts/generate_visualc_files.pl

```diff
@@ -153,6 +167,7 @@ sub gen_main_file {
     my $out = slurp_file( $main_tpl );
     $out =~ s/SOURCE_ENTRIES\r\n/$source_entries/m;
     $out =~ s/HEADER_ENTRIES\r\n/$header_entries/m;
+    $out =~ s/INCLUDE_DIRECTORIES\r\n/$include_directories/g;
```

Comment:
Should this be `/g` or `/m`? The above lines use `/m`.

Reply:
This should be /g, since there are multiple places where INCLUDE_DIRECTORIES is used, and only one place where SOURCE_ENTRIES or HEADER_ENTRIES is used.